cloudbuild.yaml

```diff
@@ -0,0 +1,24 @@
+# See https://cloud.google.com/cloud-build/docs/build-config
+
+# this must be specified in seconds. If omitted, defaults to 600s (10 mins)
+timeout: 1200s
```

Comment:
How is the timeout used?
Is this PR a WIP or ready to test?

Reply:
(AIUI) it's the overall elapsed time within which the build must complete in order to be treated as a success

Reply:
that's correct @sftim
there's a 10 minute default as per the comment, it's common to increase that for larger builds etc.